server/szurubooru/func/posts.py

```diff
@@ -429,6 +429,42 @@ def _sync_post_content(post: model.Post) -> None:
         generate_post_thumbnail(post)
 
 
+def generate_alternate_formats(post: model.Post, content: bytes) \
+        -> List[Tuple[model.Post, List[model.Tag]]]:
+    assert post
+    assert content
+    new_posts = []
+    if mime.is_animated_gif(content):
+        tag_names = [tag_name.name for tag_name in
+                     [tag.names for tag in post.tags]]
```

Comment:
Indentation

        tag_names = [
            tag_name.name
            for tag_name in [
                 tag.names
                 for tag in post.tags]]

or

        tag_names = [
            tag_name.name
            for tag_name in [tag.names for tag in post.tags]]